api/lib/infrastructure/adapters/target-profile-adapter.js

```diff
@@ -1,11 +1,17 @@
 const TargetProfile = require('../../domain/models/TargetProfile');
+const Stage = require('../../domain/models/Stage');
 const skillAdapter = require('./skill-adapter');
 
 module.exports = {
 
   fromDatasourceObjects({ bookshelfTargetProfile, associatedSkillAirtableDataObjects }) {
 
     const skills = associatedSkillAirtableDataObjects.map(skillAdapter.fromAirtableDataObject);
+    const hasStages = bookshelfTargetProfile.related('stages') && bookshelfTargetProfile.related('stages').models;
+    const stages = hasStages ?
+      bookshelfTargetProfile.related('stages').models.map((stage) => {
+        return new Stage(stage.attributes);
+      }) : [];
```

Comment:
On pourrait utiliser une variable pour stocker le résultat de `bookshelfTargetProfile.related('stages')` : 

```suggestion
    const targetProfileStages = bookshelfTargetProfile.related('stages')
    const hasStages = targetProfileStages && targetProfileStages.models;
    const stages = hasStages ?
      targetProfileStages.models.map((stage) => {
        return new Stage(stage.attributes);
      }) : [];
```